rts/Lua/LuaSyncedRead.cpp

```diff
@@ -7071,7 +7113,7 @@ int LuaSyncedRead::GetFeaturePieceCollisionVolumeData(lua_State* L)
  *
  * @function Spring.GetProjectilePosition
  * @param projectileID integer
- * @return number? posX
+ * @return number|nil posX
```

Comment:
Same thing, prefer the question mark version. Add question mark for returns below (when projectile couldn't be found)